lib/ddtrace/transport/trace_formatter.rb

```diff
@@ -155,6 +156,15 @@ def tag_sample_rate!
         )
       end
 
+      def tag_sampling_mechanism!
+        return unless trace.sampling_mechanism
+
+        root_span.set_tag(
+          Tracing::Metadata::Ext::Distributed::TAG_DECISION_MAKER,
+          "-#{trace.sampling_mechanism}"
```

Comment:
Here's the formatting of the "two" part tag: `part1-sampling_mechanism`, where `part1` is temporarily not populated.